api/types/state.go

```diff
@@ -29,6 +29,30 @@ const (
 	ConfigurationApplyFailed             ConfigurationState = "ApplyFailed"
 	ConfigurationDestroyFailed           ConfigurationState = "DestroyFailed"
 	ConfigurationReloading               ConfigurationState = "ConfigurationReloading"
+	GeneratingOutputs                    ConfigurationState = "GeneratingOutputs"
```

Comment:
```suggestion
	GeneratingOutputs                    ConfigurationState = "GeneratingTerraformOutputs"
```
This looks more clear.